schema/url.schema.json

```diff
@@ -3,5 +3,5 @@
+    "format": "uri-reference"
```

Comment:
FYI, see https://github.com/w3c/pub-manifest/pull/85 for why we changed these to regex. In a nutshell, we support iri-reference) but only uri-reference works) with ajv, and neither limits the scheme to http(s). the regex solution isn't ideal, but with 2019-09 of json schema it looks like format is being discouraged altogether so this may have to migrate out to another validation layer anyway.